src/Billing/BillingDetails/BillingDetails.js

```diff
@@ -215,8 +239,8 @@ const BillingDetails = (props) => {
                             </button>
                             <Form onSubmit={handleSubmit} onChange={handleChange}>
                                 <Modal show={show} size="lg"
-                                       aria-labelledby="contained-modal-title-vcenter"
-                                       centered>
+                                    aria-labelledby="contained-modal-title-vcenter"
+                                    centered>
                                     <Modal.Header>
                                         <Modal.Title>Creating new card</Modal.Title>
                                     </Modal.Header>
```

Comment:
It is good practice to format code.

Reply:
I have just clicked Shift+Alt+F in BillingDetails.js using Visual Studio code, but nothing changed, that must be the format of IDE